kolibri/plugins/learn/assets/src/views/total-points/index.vue

```diff
@@ -19,7 +19,8 @@
   module.exports = {
     $trNameSpace: 'totalPoints',
     $trs: {
-      totalPoints: 'You have earned { points, number } points!',
+      totalPoints: 'Total points',
```

Comment:
Is this used anywhere? I don't see a `$tr('totalPoints')` anywhere in the template. In any case, if it is, it might be worth giving it a more semantic name, per our discussion on aiding translators.